README.md

```diff
@@ -65,7 +254,7 @@ User installs can utilize the zip installer for Linux
 |---------------|-----------------|------|
 | Linux (x86_64)| ZIP             | https://github.com/jfrog/log-analytics/raw/master/fluentd-installer/fluentd-1.11.0-linux-x86_64.tar.gz |
 
-Download it to a directory the user has permissions to write such as the `$JF_PRODUCT_DATA_INTERNAL` locations discussed above:
+Download it to a directory the user has permissions to write such as the `$JF_PRODUCT_DATA_INTERNAL` locations discussed above in the [Environment Configuration](#environment-configuration) section.
```

Comment:
instead of this just as them to run the export command again.. so that they dont have to go back to the environment section.

Reply:
This way, we'll have the export step for every platform for both configuration and installation which is redundant

Reply:
That is the reason we added setting env variables as first step

Reply:
Please use the env var